devops/resources/delete-merged-git-branches.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+WHITELIST="open-release|origin/release$|olive|mckinsey"
+
+for branch in $(git branch -r --merged origin/master | grep -v HEAD | grep -v origin/master | grep -vE "${WHITELIST}"); do
```

Comment:
Can you add some comments in this file to explain what all the git commands are doing?  Not everyone will know that this is listing all branches that have been merged to origin/master for example.